File: ClassesDoJogo/classedojogo.js
class hero {
    constructor(nome, idade, tipo){
        this.nome = nome
        this.idade = idade
        this.tipo = tipo
    }

    atacar(){
        let ataque = ""
        if (this.tipo === "mago"){
            ataque = "magia"    
        }
        if (this.tipo === "guerreiro"){
            ataque = "espada"    
        }
        if (this.tipo === "monge"){
            ataque = "artes marciais"    
        }
        if (this.tipo === "ninja"){
            ataque = "shuriken"    
        }
        console.log(`O ${this.tipo} ${this.nome} atacou usando ${ataque}`)
    }
}
let heroi = new hero("ronaldo",35,"guerreiro")
heroi.atacar()

File: ClassesDoJogo/classedojogo.py
class hero:
    def __init__(self, nome, idade, tipo):
        self.nome = nome
        self.idade = idade
        self.tipo = tipo

    def atacar(self):
        ataque = ""
        if (self.tipo == "mago"):
            ataque = "magia"    
        
        if (self.tipo == "guerreiro"):
            ataque = "espada"    
        
        if (self.tipo == "monge"):
            ataque = "artes marciais"    
        
        if (self.tipo == "ninja"):
            ataque = "shuriken"    
        
        print(f'O {self.tipo} {self.nome} atacou usando {ataque}')
    

heroi = hero("Ronaldo",35,"guerreiro")
heroi.atacar()